js/main.js

```diff
@@ -0,0 +1,29 @@
+var photoPreview = document.querySelector('#photo-url');
+var placeHolder = document.querySelector('.placeholder');
+var $form = document.querySelector('.journal-form');
+
+function showPic(event) {
+  placeHolder.setAttribute('src', event.target.value);
+}
+
+photoPreview.addEventListener('input', showPic);
+
+data.nextEntryId = 0;
+
+function storeValues(event) {
+  event.preventDefault();
+  var titleValue = $form.elements.title.value;
+  var urlValue = $form.elements.url.value;
+  var notesValue = $form.elements.notes.value;
+  var fieldEntries = { title: titleValue, url: urlValue, notes: notesValue, entryId: data.nextEntryId };
+  data.entries.unshift(fieldEntries);
+
+  if (data.entries.length++) {
```

Comment:
Why are you trying to increment the length of the array in a conditional?? 😨 

You can delete this conditional, just keep the code inside of it.